### DISCUSSION TITLE
Unable to re-apply for Github enterprise

### DISCUSSION THREAD

**Discussion by RiddhimanRaut** [community/community#88604] [Enterprise]
### Hi everyone,

Hello all. I'm unable to reapply for Github Enterprise even though my email is verified. I did click on the user type (I'm a student), but still I'm unable to go through.

Any help would be greatly appreciated. 

![image](https://github.com/community/community/assets/40350551/33a4fee8-c57f-49af-a6e1-3afcab2f799b)

**Comment by dwaitbhatt** [comment]
I am also facing the same issue while re-verifying my academic affiliation and joining Global Campus.
